testbed/core/tests/conftest.py

```diff
@@ -8,7 +9,50 @@
     LikeActivityFactory,
     FollowActivityFactory,
 )
-from testbed.core.models import Actor
+from testbed.core.models import Actor, User
+from testbed.core.utils.actor_utils import populate_source_actor_outbox
+
+# Helper function to create an isolated actor (no signals triggered)
+def create_isolated_actor(username_prefix, role=None):
+    # Creates an actor without triggering signals for additional objects
+    role = role or Actor.ROLE_SOURCE  # Default to source
+    user = UserOnlyFactory(username=f"{username_prefix}_user")
```

Comment:
Because saving a user will trigger the create actor signal, does this result in a user with 3 actors? the tests would still pass, of course.

We might want the models to be able to handle an Actor without a User.  Then tests can just instantiate an Actor quickly and test Actor functionality.  Many tests run faster if you instantiate fewer objects because there's less likelihood that some dependency between them makes them actually save to the test db -- and objects not saved to the test db result in a much faster test.